2022-11-24 22:25:24,119 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-24 22:25:27,794 ERROR: Exception on /post_detail/2 [GET] [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\app.py:1741]
Traceback (most recent call last):
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\main\routes.py", line 131, in post_detail
    return render_template('main/post_detail.html', title="Post detail", post=post, \
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\templating.py", line 147, in render_template
    return _render(app, template, context)
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\templating.py", line 130, in _render
    rv = template.render(context)
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\jinja2\environment.py", line 1301, in render
    self.environment.handle_exception()
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\jinja2\environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\templates\main\post_detail.html", line 2, in top-level template code
    {% import 'bootstrap/wtf.html' as wtf %}
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\templates\base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 1, in top-level template code
    {% block doc -%}
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 4, in block 'doc'
    {%- block html %}
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 20, in block 'html'
    {% block body -%}
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 23, in block 'body'
    {% block content -%}
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\templates\base.html", line 49, in block 'content'
    {% block web_content %}{% endblock %}
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\templates\main\post_detail.html", line 24, in block 'web_content'
    {% if post.status <= 0.5 %}
TypeError: '<=' not supported between instances of 'str' and 'float'
2022-11-24 22:26:18,052 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-24 22:26:31,098 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-24 22:26:38,224 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-24 22:29:02,759 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-24 22:29:57,926 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-24 22:34:16,192 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-24 22:34:47,079 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-24 22:35:30,721 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-24 22:35:56,311 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-24 22:37:33,249 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-24 22:38:12,575 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-24 22:38:31,284 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-24 22:41:33,807 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-25 01:17:24,706 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-25 01:17:53,157 ERROR: Exception on /selled/1 [GET] [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\app.py:1741]
Traceback (most recent call last):
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\main\routes.py", line 167, in selled
    posts = user.selled().paginate(page=page, \
AttributeError: 'Query' object has no attribute 'selled'
2022-11-25 01:18:51,081 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-25 01:20:45,740 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-25 01:21:39,326 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-25 01:29:00,694 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-25 01:29:44,912 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-25 01:35:36,766 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-25 01:36:16,933 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-25 01:41:37,101 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-25 01:41:39,837 ERROR: Exception on /post_detail/6 [GET] [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\app.py:1741]
Traceback (most recent call last):
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\main\routes.py", line 131, in post_detail
    comments = post.comments.order_by(Comment.timestamp.desc()).paginage(
AttributeError: 'Query' object has no attribute 'paginage'
2022-11-25 01:42:38,524 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
2022-11-25 01:42:46,689 ERROR: Exception on /post_detail/9 [GET] [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\app.py:1741]
Traceback (most recent call last):
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\web_env\lib\site-packages\flask\app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\main\routes.py", line 131, in post_detail
    comments = posts.comments.order_by(Comment.timestamp.desc()).paginage(
AttributeError: 'Query' object has no attribute 'paginage'
2022-11-25 01:43:28,799 INFO: Website startup [in D:\TranXuanThuy\Project\ChiaSeDoCuWeb\website\__init__.py:67]
